source/sysadmins-guide/varnish-setup/index.md

```diff
@@ -242,10 +242,8 @@ sub vcl_recv {
         return (pass);
     }
 
-    # Do not cache these paths.
-    if (req.url ~ "^/backend" ||
-        req.url ~ "^/backend/.*$") {
-
+    # Always pass these paths directly to php without caching
```

Comment:
Please add a notice that the condition does not work with virtual urls and has been to be changed